dispatcher/DAQController.py

```diff
@@ -274,16 +274,22 @@ def ControlDetector(self, command, detector, force=False):
 
         if (dt > self.timeouts[command] and dt_last > self.time_between_commands) or force:
             run_mode = self.goal_state[detector]['mode']
-            if command in ['start','arm']:
+            if command == 'arm':
                 readers, cc = self.mongo.GetHostsForMode(run_mode)
+                hosts = (cc, readers)
                 delay = 0
+            elif command == 'start':
+                readers, cc = self.mongo.GetHostsForMode(run_mode)
+                hosts = (readers, cc) # we want the cc to delay by 1s
```

Comment:
I believe there is a reason, but I really cannot find it: why did you change the order of the hosts here?

Reply:
If we specify a delay between when hosts get this command, (https://github.com/AxFoundation/redax/blob/master/dispatcher/MongoConnect.py#L420), then whichever hosts are listed second receive the command second. For START commands we want the readers to get the command before the CC, and for STOP it's the other way around.

Reply:
ok, thanks